Client/Frontend/Strings.swift

```diff
@@ -3611,6 +3611,21 @@ extension String {
             tableName: "Shopping",
             value: "Try Again",
             comment: "Button text of the error displayed in the shopping review quality bottom sheet.")
+        public static let AdjustedRatingTitle = MZLocalizedString(
+            key: "", // Shopping.AdjustedRating.Title.v118
+            tableName: "Shopping",
+            value: "Adjusted rating",
+            comment: "Title of the adjusted rating card displayed in the shopping review quality bottom sheet.")
+        public static let AdjustedRatingDescription = MZLocalizedString(
+            key: "", // Shopping.AdjustedRating.Description.v118
+            tableName: "Shopping",
+            value: "Unreliable reviews removed",
+            comment: "Description adjusted of the rating card displayed in the shopping review quality bottom sheet.")
+        public static let AdjustedRatingStarsAccessibilityLabel = MZLocalizedString(
+            key: "", // Shopping.AdjustedRating.StarsAccessibilityLabel.v118
+            tableName: "Shopping",
+            value: "%.1f out of 5 stars",
```

Comment:
Just a question; I am not sure our string export script will support `%.1f`, nor what this will look like for translators. Do we have some other equivalent in our codebase/strings.file which we could compare this with?

Reply:
I took a look and it seems it's the first time we are using floating point numbers in strings. (previously we used %d for numbers) the .1 is necessary because by default it will show like 5 point decimals which is not what we want. I will check too see if there's another alternative.

Reply:
Is there a way to test locally that it fails? I'm not familiar with the export mechanism.

Reply:
@razvanlitianu could we just use `%@` and convert the number in the right format to a string? Not sure that is a good idea - @Delphine maybe you can weight in?

Also could you please adjust the comment? It seems there is a word missing in the second sentence to make sense. And it would be good to include where this is used for context for the translators.

Reply:
One solution that works was to pre convert the number to a String and use %@. I updated the PR